internal/bootstrap/bootstrap_tpl.go

```diff
@@ -112,10 +112,21 @@ type BootstrapTplArgs struct {
 	// the envoy_prometheus_bind_addr listener.
 	PrometheusScrapePath string
 
-	PrometheusCAFile   string
-	PrometheusCAPath   string
+	// PrometheusCAFile is the path to a CA file for Envoy to use when serving TLS on the Prometheius metrics
+	// endpoint. Only applicable when envoy_prometheus_bind_addr is set in the proxy config.
+	PrometheusCAFile string
+
+	// PrometheusCAPath is the path to a directory of CA certificates for Envoy to use when serving the Prometheus
+	// metrics endpoint. Only applicable when envoy_prometheus_bind_addr is set in the proxy config.
+	PrometheusCAPath string
+
+	// PrometheusCertFile is the path to a certificate file for Envoy to use when serving TLS on the Prometheus
+	// metrics endpoint. Only applicable when envoy_prometheus_bind_addr is set in the proxy config.
 	PrometheusCertFile string
-	PrometheusKeyFile  string
+
+	// PrometheusKeyFile is the path to a private key file Envoy to use when service TLS on the Prometheus metrics
```

Comment:
```suggestion
	// PrometheusKeyFile is the path to a private key file Envoy to use when serving TLS on the Prometheus metrics
```

(Now that these comments are added into the consul repo, you should be able to run `make copy-bootstrap-config` to fix this.)

Reply:
Done!